saleor/settings.py

```diff
@@ -26,6 +28,8 @@
 from .core.languages import LANGUAGES as CORE_LANGUAGES
 from .core.schedules import initiated_sale_webhook_schedule
 
+django_stubs_ext.monkeypatch()
```

Comment:
That's unnecessary code running at startup
```suggestion
if TYPE_CHECKING:
    django_stubs_ext.monkeypatch()
```

Reply:
It is not.